eclair-core/src/main/resources/reference.conf

```diff
@@ -135,6 +135,7 @@ eclair {
     // expiry-delta-blocks.
     fulfill-safety-before-timeout-blocks = 24
     min-final-expiry-delta-blocks = 30 // Bolt 11 invoice's min_final_cltv_expiry; must be strictly greater than fulfill-safety-before-timeout-blocks
+    max-restart-watch-delay = 60 seconds // we add a random delay before watching funding transactions after restart
```

Comment:
Nit: we use a different naming for the same use-case elsewhere (`router.watch-spent-window`).

Reply:
True, but then we would be inconsistent with the other two values right below, which are used in the same way...